### PR TITLE
7.x solr thumbnail

### PR DESCRIPTION
**Pull request by MorganDawe**
**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-1870)

# What does this Pull Request do?

Basic fix for SPARQL and SOLR thumbnail display width.

# What's new?
Add a max-width to thumbnails using the SOLR Backend display

Example:
* Added .islandora-object img max-width rule

# How should this be tested?

* Add a large .png as a thumbnail for an object (large as in 'width')
* View object in a collection grid using the SOLR backend display (maintains aspect)
* View object in a collection grid using SPARQL Legacy display (maintains aspect) 

# Note

Related in part to this ticket: https://jira.duraspace.org/browse/ISLANDORA-1609, i believe the ultimate solution is to ensure both displays are run thru the same preprocess functions, using the same HTML and CSS as each other. However, that may be to dramatic of a change outside of a major release. 

# Interested parties
Tag @Islandora/7-x-1-x-committers

### CODE REVIEW

File: css/islandora.objects.css [hunk 1]
```diff
@@ -10,6 +10,17 @@ div.islandora-objects > div {
   display: inline-block;
 }
 
+/* img styling (max-width) - Setting a max width to ensure a thumbnails width
+ * does not exceed that of its parent container (ex: 20%).
+ *
+ * img styling (max-height) - Avoid making assumptions on max height here, allowing auto
+ * height based on max-width of 100%. Consider overrideing this file in your theme,
```
Comment (adam-vessey) [inline css/islandora.objects.css]: `s/overrideing/overriding/`

Comment (MorganDawe) [inline css/islandora.objects.css]: Me fail English, that's unpossible

**Comment by MorganDawe** [comment]
@Islandora/7-x-1-x-committers

**Comment by MorganDawe** [comment]
@Islandora/7-x-1-x-committers

**Comment by jordandukart** [comment]
@dannylamb

**Comment by DiegoPino** [comment]
@MorganDawe just quick, because I don't see any harm to make max with 100% (means I'm ok): does this addresses the note about unifying the CSS approach? or is this a temporary solution? I see that specific CSS islandora-object class not being used that broadly, so how this affects/fixes the problem?

**Comment by dannylamb** [comment]
@DiegoPino If you would like to see a unified approach, feel free to make a ticket in JIRA and we'll address it.

**Comment by DiegoPino** [comment]
@dannylamb thanks for your suggestion. The unified approach was discussed in the comitters call as the way to go as noted here https://jira.duraspace.org/browse/ISLANDORA-1870?focusedCommentId=53300&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-53300

**Pull request merged**